dask/bytes/core.py

```diff
@@ -162,15 +162,18 @@ class OpenFile(object):
         The encoding to use if opened in text mode.
     errors : str or None, optional
         How to handle encoding errors if opened in text mode.
+    newline : None, '', '\n', '\r', or '\r\n'.
+        Passed to TextIOWrapper in text mode
     """
     def __init__(self, fs, path, mode='rb', compression=None, encoding=None,
-                 errors=None):
+                 errors=None, newline=None):
         self.fs = fs
         self.path = path
         self.mode = mode
         self.compression = get_compression(path, compression)
         self.encoding = encoding
         self.errors = errors
+        self.newline = newline
```

Comment:
Does this need to be validated that it's one of the valid newline options? What do Python / pandas do here?

Reply:
I just tested, and Python raises a `ValueError`. I see that no other parameters passed to `open_files`/`OpenFile` get validated at initialization, so I'm thinking the consistent thing to do is to pass it in as-is, and let the exception get raised when the call to `io.TextIOWrapper` is made. If you want, though, I can add code to check it at initialization.

Reply:
Letting Python handle it when we use it later on seems just fine.